src/Core/Prefab/GlobeView.js

```diff
@@ -101,6 +102,8 @@ function GlobeView(viewerDiv, coordCarto, options = {}) {
     let positionCamera;
     if (coordCarto instanceof Coordinates) {
         positionCamera = coordCarto.as('EPSG:4326');
+    } else if (coordCarto.coord instanceof Coordinates) {
+        positionCamera = coordCarto.coord.as('EPSG:4326');
```

Comment:
I meant to be able to use the all properties `cameraTransformOptions`  (tilt, heading...). This request may be a modification in  `GlobeControls`

Reply:
This needs more changes than the initial intent of this PR, as there isn't always a `GlobeControls` (if `options.noControls === true`). What about doing it later ?

Reply:
Yes, you can remove this change

if `options.noControls === true` you can use `CameraUtils` instead of controls